setup.py

```diff
@@ -77,7 +57,17 @@ def get_packages(package):
         'Programming Language :: Python :: 3.4',
         'Topic :: Utilities',
     ],
+    # install_requires=['cryptography'],
     test_suite='tests',
-    tests_require=['pytest', 'pytest-cov'],
-    cmdclass={'test': PyTest},
```

Comment:
Why are we removing PyTest from here? Won't this break our existing `setup.py test` functionality?